README.md

```diff
@@ -1,108 +1,138 @@
-# GDS Metrics: Dropwizard version
-Library for [prometheus](https://prometheus.io/) instrumentation in [Dropwizard](http://www.dropwizard.io) based apps.
+# GDS metrics for Dropwizard apps
 
-## Overview
+GDS Metrics are in Alpha and these instructions are subject to change.
 
-The library can be added to your web app to capture metrics about how it's performing. These metrics are served from an endpoint on your app and can be scraped by Prometheus and turned into Grafana dashboards.
+GDS [Dropwizard][] metrics enable your Java web app to export performance data to [Prometheus][] and can be added to your app using this library. Once you’ve added the library, metrics data is served from an endpoint on your app and is scraped by Prometheus. This data can be turned into performance dashboards using [Grafana][].
 
-## Build the project
-`./gradlew build`
+You can read more about the Reliability Engineering monitoring solution [here][].
 
-## Publish to a Maven local repositoty
-`gradle publishToMavenLocal`
+## Before using GDS metrics
 
-## Adding the library to your project
+Before using GDS metrics you should have:
 
-Add the online repository where the library is stored.
+* created a Java app
+* deployed that Java app to [GOV.UK Platform as a Service (PaaS)][]
 
-### Maven
+## How to build your project
 
-```
-<repositories>
-    <repository>
-        <id>reliability-engineering-repository</id>
-        <name>Repository containing reliability enginieering dependencies</name>
-        <url>https://dl.bintray.com/reliability-engineering-gds/gds_metrics_dropwizard</url>
-    </repository>
-</repositories>
-``` 
-
-### Gradle
-```
-repositories {
-    maven { url "https://dl.bintray.com/reliability-engineering-gds/gds_metrics_dropwizard" }
-}
-```
+To use GDS metrics you must:
 
-Add the library as dependency to your project.
+1. Run the following [Gradle][] command to build your project:
 
-### Maven
-```
-<dependency>
-    <groupdId>engineering.gds-reliability</groupdId>
-    <artifactId>gds-metrics-dropwizard</artifactId>
-    <version>1.0.0</version>
-</dependency>
-```
+    `./gradlew build`
```

Comment:
Will someone using this already have the Gradle build tool installed?

Reply:
it doesn't matter, it should Just Work™.  (`./gradlew` calls a gradle wrapper script in this repository, which downloads a local copy of gradle to run)